### PR TITLE
Add guide on how to build demos for the Flax sprint

### PR DESCRIPTION
**Pull request by osanseviero**
Add a guide on how to build a demo for the Flax sprint. This guide will be also updated early next week, but I would like to submit this version on Friday.

Note: this will be extended more once the evaluation is a bit more defined and with the launch of S.

### CODE REVIEW

File: examples/research_projects/jax-projects/README.md [hunk 1]
```diff
@@ -977,9 +980,84 @@ For more information, check out [this PR](https://github.com/huggingface/hugging
...
+* **Fill Mask:** Predict potential words for a mask token. [Example](https://huggingface.co/bert-base-uncased?).
+* **Question Answering:** Given a context and a question, predict the answer. [Example](https://huggingface.co/bert-large-uncased-whole-word-masking-finetuned-squad).
+* **Sentence Simmilarity:** Predict how similar a set of sentences are. Useful for Sentence Transformers.
+* **Summarization:** Given a text, output a summary of it. [Example](https://huggingface.co/sshleifer/distilbart-cnn-12-6).
+* **Table Question Answering:** Given a table and a question, predict the answer. [Example](https://huggingface.co/google/tapas-base-finetuned-wtq).
+* **Text Generation:** Generate text based on a prompt. [Example](https://huggingface.co/gpt2)
+* **Token Classification:** Useful for tasks such as Named Entity Recognition and Part of Speech. [Example](https://huggingface.co/dslim/bert-base-NER).
+* **Zero-Shot Classification:** Too cool to explain with words. Here is an [example](https://huggingface.co/typeform/distilbert-base-uncased-mnli)
```
Comment (LysandreJik) [inline examples/research_projects/jax-projects/README.md:1001]: "Too cool to explain with words" lol

**Review by LysandreJik — APPROVED** [review]
Looks great!

**Comment by osanseviero** [comment]
Some TODO ideas (for a later iteration)
* Add small guide on Streamlit
* Add small guide on Gradio
* Add small guide on how to use S
* Explain why the importance of a demo

**Comment by osanseviero** [comment]
I'll merge this today. I'll do a follow up PR on Monday extending the demo section.

**Pull request merged**